### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by ghost** [hardcore-sushi/DroidFS#24] [Q&A]
1. Can I Use Other File Explorer?
2. Non Root Yes?
3. What Is Caveats When I Use DroidFS?

**Comment by hardcore-sushi** [comment]
For now, volumes are only accessible in plaintext through the internal DroidFS file explorer.
DroidFS works without root access.
The app has not been audited so it shouldn't be considered absolutely secure. Moreover, some security risks are presents on Android (apps with root access, screen-recorders, malicious keyboard, unlocked bootloader, ROM backdoors, firmware backdoors etc.)

**Comment by ghost** [comment]
> For now, volumes are only accessible in plaintext through the internal DroidFS file explorer.
> DroidFS works without root access.
> The app has not been audited so it shouldn't be considered absolutely secure. Moreover, some security risks are presents on Android (apps with root access, screen-recorders, malicious keyboard, unlocked bootloader, ROM backdoors, firmware backdoors etc.)

ayeee bro please open discussion tabs and move this issues to discussion tab
